data/campaigns/Eastern_Invasion/ART_MUSIC_LICENSE.txt

```diff
@@ -0,0 +1,383 @@
+
+AUDIO
+===============
+
+MUSIC
+---------------
+
+cry from elensefar (pluft) - CC BY-SA 4.0
+    https://forums.wesnoth.org/viewtopic.php?t=48565
+    https://soundcloud.com/pluft/cry-from-elensefar
+
+heavens (gabriel silver), remixed by dalas - CC BY-SA 4.0 (both)
+    https://forums.wesnoth.org/viewtopic.php?t=48565
```

Comment:
That's Pluft's thread, Gabriel's is https://r.wesnoth.org/t51483 .

This attribution notice probably doesn't meet the requirements of CC BY-SA 4.0. Gabriel has said that he used the Vienna Symphonic Library, has credited them, yet that credit isn't found here.

This particular work, with its drum intro, probably can't be licensed under CC BY or CC BY-SA. Thinking of the drum intro to Heavens, I'd like to sample it as a wardrums sound effect, but the currently published VSL licenses explicitly try to exclude that. I think this indicates that this particular work can't be licensed under CC BY or CC BY-SA. The VSL licenses seem restrict music using them to commercial releases, my gut feeling is that they expect the music to have standard commercial restrictions placed on its further use.

The VSL license might have been different at the time that Gabriel created the music, but the current wordings according to https://www.vsl.co.at/en/Products and https://www.vsl.co.at/de/Products (click on Terms of License at the bottom of the pages) are:

> You may not, or allow any third party to, ... samples ...
> You may use the Software on any commercial music release (including music libraries), public performance, broadcast, or similar occasion. You may use any of the Sounds in a commercial recording without paying any additional license fees. The sounds are guaranteed 100% copyright clean. Multimedia and Game Music use of the Software is restricted to use within the musical compositions themselves.

> ist es dem Lizenznehmer oder Dritten nicht gestattet, ... Samples ...
> Der Lizenznehmer darf die Software für jede kommerzielle Musikveröffentlichung (inklusive Music Libraries, Archivmusik), öffentliche Aufführung, Sendung oder für ähnliche Gelegenheiten verwenden. Die Rechte an den Klangdaten sind zu hundert Prozent für eben diese Zwecke geklärt. Die Verwendung von Vienna Instruments-Produkten erfordert eine besondere Erwähnung der Vienna Symphonic Library auf den Drucksachen oder in sonstigen Begleittexten der Musikveröffentlichung: „Enthält Orchester-Samples der Vienna Symphonic Library“ oder „Orchestral samples included in this recording from the Vienna Symphonic Library“. Die Erwähnung im Vor- oder Abspann von Filmen ist erwünscht, aber nicht Bedingung. Die Verwendung der Software in Multimedia-Projekten und bei Computer- und Videospielen darf über die Erstellung der Musik nicht hinausgehen.

Those English and German texts probably ought to be the same. The extra bit in German is about having to put an attribution notice in any accompanying liner notes.

Obligatory: I'm not a lawyer, this isn't legal advice, I'd rather reject stuff out of caution.

Reply:
> That's Pluft's thread, Gabriel's is https://r.wesnoth.org/t51483 .
> Gabriel has said that he used the Vienna Symphonic Library, has credited them, yet that credit isn't found here.

Apologies on both of these; mistakes by me.  And an additional mistake that I realized while thinking about the song: some sound effects were also pulled from Gabriel's "Waiting for Dawn", which I forgot about when adding to the license file. https://youtu.be/_5effk-4GmU

> This particular work, with its drum intro, probably can't be licensed under CC BY or CC BY-SA.

That would be unfortunate, but I understand if it's necessary.  I didn't completely understand your explanation - is it something I could work around by removing certain sounds, like cutting out the drums intro?  Or was that an example to show why the original VSL license would have prohibited any alternations to the original song?

Reply:
I understand the VSL licence as that you cannot publish the samples/sounds (they specifically say you cannot publish dls sets) themselves but you can publish music you produced using them. I mean that is more or less the point, no?

I'm pretty sure most if not all our music is done with commercial sound libraries similar to VSL.
VSL is even specifically linked here: https://r.wesnoth.org/t34862

Reply:
This has made my head hurt, sorry for the unclear explanation that I gave earlier. I've emailed VSL and asked if the word "commercial" in their license is an intentional restriction.

Cutting out the drums intro would fix one of the issues that I believe exists. Looking at the other two sound-software companies that soliton- has linked to, EastWest has a clause about not leaving a sound easily rippable: "the Audio Loops must be used in a musical context with at least two other instruments that contribute significantly to the composition; and (2) The entire Audio Loop cannot be left exposed at any time in the composition".

The other issue that I believe exists with VSL's terms is the word "commercial" in "You may use the Software on any commercial music release (including music libraries), public performance, broadcast, or similar occasion." I'm wondering if it's an intentional restriction against non-profit releases.

Reply:
Okay, I was worrying too much. Here's the answer from VSL:

> > Reading the Terms of License on your website, it seems that the English and German texts aren't equivalent, with the German one contradicting its own FAQ. The German FAQ's answer to "Muss ich die Verwendung der Sounds in den Credits angeben?" is the same as the English one, yet section 5 of the German license has two extra sentences, making that FAQ's answer only fit the English license.
> >
> > The way that the website treats mouse-drags on the license window as a request to scroll up and down prevents highlighting text from the license. This makes it hard to copy a quote into the rejection text.
> >
> > As a developer on a game that's a hobby project written by volunteers, I'm wondering if the VSL license is a blanket ban on contributing to hobby projects, because of the adjective "commercial" in section 5; please would you say whether that's the intention? The composition that I'm currently evaluating will be rejected in any case, because the first couple of bars of the music are a slow drum beat, our hobby project only accepts art under the CC BY-SA 4.0 license, and if I believed the song really was under that CC license then I'd sample it as a sound effect. However, it would save a lot of time to know whether the VSL is a blanket ban even when multiple instruments are playing in a way that they can't be isolated.
>
> Thank you for pointing us to this oversight. The Terms were adapted a year ago, with the requirement to mention VSL removed, and this was overlooked in section 5 of the German version. I have just fixed this.
>
> Regarding your other question, thank you for asking! Please go ahead and use our samples for the game. Usually questions on the commercial use of our samples come up, that's why it is specifically mentioned here. Of course you can use our sounds and software for any hobby projects. If there are a few bars with isolated drums that's fine, no problem. It would only get problematic if a track offers a lot of single separately available samples that a third party could use for re-sampling. It doesn't seem to be the case in your project.